libbeat/autodiscover/providers/kubernetes/config.go

```diff
@@ -61,5 +69,30 @@ func (c *Config) Validate() error {
 		return fmt.Errorf("no configs or hints defined for autodiscover provider")
 	}
 
+	// Check if host is being defined and change it to node instead.
+	if c.Node == "" && c.Host != "" {
+		c.Node = c.Host
+		logp.L().Warnf("`host` will be deprecated in 8.0. use `node` instead")
```

Comment:
:heart: Could you use `cfgwarn.Deprecate` deprecate here? Also, it would be nice to update our manifests to avoid this warning to new users: https://github.com/elastic/beats/tree/master/deploy/kubernetes

Reply:
We should make this change also in `add_kubernetes_metadata`

Reply:
will add this as part of follow up PR to add scope setting